src/parser.rs

```diff
@@ -642,6 +645,12 @@ fn parse_simple_pseudo_class(context: &ParserContext, name: &str) -> Result<Simp
     }
 }
 
+fn parse_lang_pseudo_class(input: &mut Parser) -> Result<SimpleSelector, ()> {
+    let string = try!(input.expect_ident()).into_owned();
```

Comment:
Please remove `.into_owned()` here, it does an unnecessary memory allocation. (`&Cow<str>` auto-dereferences to `&str` in the `from_slice` call.)